pyheaders/compiler.py

```diff
@@ -157,6 +157,7 @@ def __init__(self, *, commands_path: AnyStr = None, exclude_flags: Dict[Text, in
         self.exclude = exclude_flags
 
     @staticmethod
+    @lru_cache
```

Comment:
What's the point? is it not only running once per program?

Reply:
Once per processed file, when working on a project, it will be called for every file.

Reply:
Once per processed file, when working on a project, it will be called for every file.